assets/js/src/git-file-manager.js

```diff
@@ -109,6 +116,15 @@ class YamlWriter extends FileWriter {
     });
   }
 
+  mergeFirstLevel(oldObject, newObject, propPath) {
+    if (newObject[propPath]) delete newObject[propPath];
+    if (oldObject[propPath]) delete oldObject[propPath];
+
+    Object.assign(oldObject, newObject);
+
+    return newObject;
+  }
+
```

Comment:
Basically what it does is remove the propPath element (since we don't want to update it), merge the values of the new object on the values of the old object and return that.

The returned value doesn't have the propPath element, but it does work since we're adding the merged one on line 107/114 anyway.